google/resource_dns_managed_zone.go

```diff
@@ -72,7 +78,18 @@ func resourceDnsManagedZoneCreate(d *schema.ResourceData, meta interface{}) erro
 		Description: d.Get("description").(string),
 	}
 
+	if clabels, ok := d.GetOk("labels"); ok {
+		labels := map[string]string{}
```

Comment:
we actually have a util called `expandLabels` already, so you may as well just use that